internal/http/image.go

```diff
@@ -37,27 +38,35 @@ func NewImageHandler(
 	}
 }
 
-// ServeHTTP sends message with client`s image to WhatsApp server.
-func (h *SendImageHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+func (h *SendImageHandler) Handle(w http.ResponseWriter, r *http.Request) *AppError {
 	decoder := json.NewDecoder(r.Body)
 	var msgReq SendImageRequest
 	err := decoder.Decode(&msgReq)
 	if err != nil {
-		handleError(w, "can't decode request", err, http.StatusBadRequest)
-		return
+		return &AppError{
```

Comment:
1) Add fields when you use structures. It's difficult to read your code with structures in Github
2) Maybe it's better to create a function-helper to create an error. But it up to you

Reply:
I think this simple operation doesn’t deserve separate function yet:)